tensorflow/contrib/makefile/download_dependencies.sh

```diff
@@ -45,7 +47,7 @@ CUB_URL="$(grep -o 'https.*cub/archive.*zip' "${BZL_FILE_PATH}" | grep -v mirror
 
 # Required for TensorFlow Lite Flex runtime.
 FARMHASH_URL="https://mirror.bazel.build/github.com/google/farmhash/archive/816a4ae622e964763ca0862d9dbd19324a1eaf45.tar.gz"
-FLATBUFFERS_URL="https://github.com/google/flatbuffers/archive/1f5eae5d6a135ff6811724f6c57f911d1f46bb15.tar.gz"
+FLATBUFFERS_URL="https://github.com/google/flatbuffers/archive/v1.9.0.tar.gz"
```

Comment:
Could you please change this back? I don't think it's related to the error.

Reply:
The original URL also uses a newer version of flatbuffer (1.10.0). It was from [Aug 20](https://github.com/google/flatbuffers/commit/1f5eae5d6a135ff6811724f6c57f911d1f46bb15). The 1.9.0 release was in April.